server/services/address.js

```diff
@@ -10,7 +10,8 @@ async function simulatedFind (inputPostcode) {
   try {
     const postcode = inputPostcode.toUpperCase()
     const part1 = postcode.split(' ')[0]
-    const filename = path.join('./server/simulated_data/', part1, postcode + '.json')
+    const filename = path.join(config.simulatedDataPath, part1, postcode + '.json')
+    console.log(filename)
```

Comment:
I appreciate this console.log is probably in for the purposes of testing but thought I'd highlight just in case.

Reply:
Ah, good spot. I'll remove.